JProfiler Update Action stopped working

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior

Action runs should complete successfully and report if a new version is available.

## Current Behavior

```
panic: no candidate version

goroutine 1 [running]:
main.main()
	./main.go:49 +0x10[6](https://github.com/paketo-buildpacks/jprofiler/actions/runs/10465616230/job/28981149562#step:7:7)
```

## Possible Solution

It's possible that the site we look at to see if there are new updates has changed. More investigation needs to be done.

## Steps to Reproduce

Run the action locally to reproduce.

--- a/actions/jprofiler-dependency/main.go
+++ b/actions/jprofiler-dependency/main.go
@@ -20,30 +20,36 @@ import (
 	"fmt"
 	"regexp"
 
-	"github.com/gocolly/colly"
+	"net/http"
 
+	"github.com/PuerkitoBio/goquery"
 	"github.com/paketo-buildpacks/pipeline-builder/actions"
 )
 
 func main() {
 	inputs := actions.NewInputs()
 
-	c := colly.NewCollector()
-
 	cp := regexp.MustCompile(`^Release ([\d]+)\.([\d]+)\.([\d]+).*$`)
 	versions := make(actions.Versions)
-	c.OnHTML("h5", func(element *colly.HTMLElement) {
-		if p := cp.FindStringSubmatch(element.Text); p != nil {
 
+	res, err := http.Get("https://www.ej-technologies.com/jprofiler/changelog")
+	if err != nil {
+		panic(err)
+	}
+	defer res.Body.Close()
+
+	doc, err := goquery.NewDocumentFromReader(res.Body)
+	if err != nil {
+		panic(err)
+	}
+
+	doc.Find("div.release-heading").Each(func(i int, s *goquery.Selection) {
+		if p := cp.FindStringSubmatch(s.Text()); p != nil { 
 			v := fmt.Sprintf("%s.%s.%s", p[1], p[2], p[3])
 
 			versions[v] = fmt.Sprintf("https://download-gcdn.ej-technologies.com/jprofiler/jprofiler_linux_%s_%s_%s.tar.gz", p[1], p[2], p[3])
 		}
 	})
-
-	if err := c.Visit("https://www.ej-technologies.com/download/jprofiler/changelog.html"); err != nil {
-		panic(err)
-	}
 
 	if o, err := versions.GetLatest(inputs); err != nil {
 		panic(err)

